core/src/com/unciv/logic/MapSaver.kt

```diff
@@ -16,22 +16,23 @@ object MapSaver {
     private fun getScenario(scenarioName:String) = Gdx.files.local("$scenariosFolder/$scenarioName")
     
     fun saveMap(mapName: String,tileMap: TileMap) {
-        getMap(mapName).writeString(Gzip.zip(json().toJson(tileMap)), false)
+        getMap(mapName).writeString(Gzip.zip(json().toJson(tileMap.stripAllUnits())), false)
```

Comment:
I disagree with stripping the units when saving/loading a map... I think people might want to create maps with units on them even not as part of a scenario.

Reply:
Ok. I will leave fix it in the next commit.